File: PRÁCTICA SEGUNDO PARCIAL/Ejercicio11/Python/SumaVectores.py
# Pedir al usuario ingresar los vectores
entrada1 = input("Ingresa los elementos del primer vector separados por espacios: ")
vector1 = [int(num) for num in entrada1.split()]

entrada2 = input("Ingresa los elementos del segundo vector separados por espacios: ")
vector2 = [int(num) for num in entrada2.split()]

# Verificar que los vectores tengan la misma longitud
if len(vector1) != len(vector2):
    print("Los vectores deben tener la misma longitud.")
else:
    resultado = []
    for i in range(len(vector1)):
        suma = vector1[i] + vector2[i]
        resultado.append(suma)
    
    print("La suma de los elementos es:", resultado)

File: PRÁCTICA SEGUNDO PARCIAL/Ejercicio11/Java/SumaVectores.java
public class SumaVectores {
    public static void main(String[] args) {
        int[] vector1 = {1, 2, 3, 4, 5};
        int[] vector2 = {6, 7, 8, 9, 10};

        int n = vector1.length; // Ambos vectores deben tener la misma longitud

        int[] resultado = new int[n];

        for (int i = 0; i < n; i++) {
            resultado[i] = vector1[i] + vector2[i];
        }

        // Imprimir el resultado
        System.out.print("La suma de los vectores es: ");
        for (int i = 0; i < n; i++) {
            System.out.print(resultado[i] + " ");
        }
    }
}
